src/test/resources/TestNG.xml

```diff
@@ -2,11 +2,14 @@
 <!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
 
 <suite name="TestNG">
-
+    <parameter name="BaseURL" value="https://qa.koel.app/"/>
     <test name="Example Test" preserve-order="false">
+        
         <classes>
             <class name="LoginTests"/>
             <class name="BaseTest"/>
+            <class name="ProfileTests"/>
```

Comment:
Please, add Homework16 in TestNG.xml to run tests

![image](https://github.com/BeatrizSmith/A47-QA-Automation-Framework/assets/104761482/b0292ccb-86c1-4852-ab17-e825549860e7)